velox/docs/functions/spark/binary.rst

```diff
@@ -0,0 +1,19 @@
+================
+Binary Functions
+================
+
+.. function:: xxhash64(x) -> integer
+
+    Computes the xxhash64 hash of x.
+
+.. function:: hash(x) -> integer
+
+    Computes the hash hash of x.
```

Comment:
What is "hash hash"?

Reply:
Changed "hash hash" to "hash". Sorry, my writing is not very idiomatic. So I made a complete fool of myself.